beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md

```diff
@@ -34,9 +34,9 @@ After you have tested your function, you can click **Upload** to save your valid
 | Key | Type | Description |
 | :--- | :--- | :--- |
 | `uuid` | String | Universally unique identifier for the event. |
-| `timestamp` | Number | Timestamp from the client when the event was sent to Project Griffon. |
+| `timestamp` | Number | Timestamp from the client when the event was sent to Assurance. |
 | `eventNumber` | Number | Used to order when the event was sent. This key is useful when events have the same timestamp. |
-| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.griffon\). |
+| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.assurance\). |
```

Comment:
Same here. Leave it as com.adobe.griffon for now.

Reply:
Sounds good. Change reverted.